src/present.cpp

```diff
@@ -227,13 +227,20 @@ namespace D3D11On12
     void Device::AcquireResource(D3D10DDI_HDEVICE hDevice, D3D10DDI_HRESOURCE, HANDLE hSyncToken) noexcept
     {
         auto pThis = CastFrom(hDevice);
+        SyncTokenExtensionData data;
+        data.hSyncToken = hSyncToken;
+        data.sharingContractPresentResource = nullptr;
         pThis->GetBatchedContext().BatchExtension(&pThis->m_AcquireResourceExt, hSyncToken);

```

Comment:
This needs to be updated to pass `&data`

Reply:
Ah, thanks for catching that